WebRoot/templates/admin/Widgets.template

```diff
@@ -35,6 +35,12 @@
 			</td>
 			<td id='${id}_settingimg'						style='width:20px'></td>
 			<td id='${id}_dropdown'	class='ZDropDown'		style='width:20px'></td>
+			<td id='loadingStatusBar' class='loadingContainer' style='visibility:hidden;'>
+				<div style='position:relative;height:inherit;'>
+					<div class='LoaderTitle'>Loading...</div>
```

Comment:
please don't set non-translated string in a template.

Reply:
Could I leave this empty? It is getting replaced with the .properties value in the toggleLoadingStatusBar method.

Reply:
I think we can pass a string in ZaMsg like 
https://github.com/Zimbra/zm-admin-console/blob/develop/WebRoot/templates/dwt/Widgets.template#L530
(it's AjxMsg, though)

either is fine; empty value or ZaMsg.BUSY_LOADING.

Reply:
Ready. Replaced with ZaMsg.BUSY_LOADING